docs/development/Offline_mode.md

```diff
@@ -1,10 +1,14 @@
 # How to Implement Offline Capabilities
 
-**Flutter resources:**    
-https://github.com/tekartik/sqflite/blob/master/sqflite/README.md  
-https://pub.dev/packages/connectivity   
-https://flutter.dev/docs/cookbook/images/cached-images   
-https://pub.dev/packages/cached_network_image
+**Flutter resources:**
+
+[SQFlite](https://github.com/tekartik/sqflite/blob/master/sqflite/README.md)  
+
+[connectivity](https://pub.dev/packages/connectivity)

```

Comment:
```suggestion
[Connectivity](https://pub.dev/packages/connectivity)

```

Reply:
It's the name of the package. You still think is should be with capital 'C'?

Reply:
No, it is fine then.